crates/meilisearch-auth/src/store.rs

```diff
@@ -80,19 +79,18 @@ impl HeedAuthStore {
 
         self.keys.put(&mut wtxn, uid.as_bytes(), &key)?;
 
-        // delete key from inverted database before refilling it.
+        // Delete existing actions
         self.delete_key_from_inverted_db(&mut wtxn, &uid)?;
-        // create inverted database.
-        let db = self.action_keyid_index_expiration;
 
+        // Create new actions with bitflags
```

Comment:
We now have bit masks. All the code below should be changed to build a bitflag mask; we don't need to collect the actions in an array anymore.

I would expect something like:
```rust
let mut mask = 0;
for action in &key.actions {
   mask |= action.as_bitflag(); // guessing the method name, I didn't check in the doc.
}
``` 

maybe in the newly added `KeyActions::new`